pages/pipelines/block_step.md.erb

```diff
@@ -18,17 +18,33 @@ steps:
 
 <%= image "block-step.png", width: 944/2, height: 364/2, alt: "Screenshot of a basic block step" %>
 
-You can add form fields to block steps with the `fields` attribute. There are two field types available: text or select. The select input type is displayed as radio buttons when it contains less than six items, and as a drop down list when it contains more than six items.
-
-The data you collect from these fields is then available to subsequent steps in the pipeline in the [build meta-data](/docs/pipelines/build-meta-data). Block steps with input fields can only be defined using a `pipeline.yml`.
+You can add form fields to block steps with the `fields` attribute. There are two field types available: text or select. The select input type is displayed as radio buttons when it contains less than six items, and as a drop down list when it contains more than six items. Block steps with input fields can only be defined using a `pipeline.yml`.
 
 <%= image "block-step-with-fields.png", width: 944/2, height: 1178/2, alt: "Screenshot of a block step with input fields" %>
 
+The data you collect from these fields is then available to subsequent steps in the pipeline in the [build meta-data](/docs/pipelines/build-meta-data). 
+
+In this example, the `pipeline.yml` defines an input step with the key `release-name`. The Bash script then accesses the value of the step by getting that key from the meta-data.
+
+```yml
+ - block: "Release"
+    prompt: "Fill out the details for release"
+    fields: 
+      - text: "Release Name"
+        key: "release-name"
+```
+{: codeblock-file="pipeline.yml"}
+
```

Comment:
Do we need to mention that the bash script isn't run *in the example*? Kinda reinforcing the next para on line 43?

Reply:
Resolving this by adding the run script step to the yml.